frontend/public/locales/en/translation.json

```diff
@@ -356,12 +356,13 @@
     "clusterPool.creation.success.message": "{{name}} was successfully created.",
     "clusterPool.creation.success.title": "Cluster pool created",
     "clusterPool.destroy": "Destroy cluster pool",
-    "clusterPool.modal.scale.input": "Set desired cluster pool size",
-    "clusterPool.modal.scale.message": "Specify the desired size of <bold>{{clusterPoolName}}</bold>. Adjusting the size of the cluster pool will result in the creation or destruction of clusters; only unclaimed clusters will be destroyed if downsizing.",
+    "clusterPool.modal.scale.message": "Specify the desired size of <bold>{{clusterPoolName}}</bold>. Adjusting the size or running count of the cluster pool will result in the creation or destruction of clusters; only unclaimed clusters will be destroyed if downsizing.",
     "clusterPool.modal.scale.poolName": "Cluster pool name",
     "clusterPool.modal.scale.poolNamespace": "Cluster pool namespace",
+    "clusterPool.modal.scale.runningCount.input": "Set desired number of running clusters for this cluster pool.",
+    "clusterPool.modal.scale.size.input": "Set desired cluster pool size",
     "clusterPool.modal.scale.title": "Scale cluster pool",
-    "clusterPool.modal.scale.validation.greaterThanZero": "clusterPool.modal.scale.validation.greaterThanZero",
+    "clusterPool.modal.scale.validation.greaterThanZero": "The cluster pool size and running count must both be greater than 0.",
```

Comment:
Your validation checks for less than 0. Equal to 0 is valid for both pool size and running count.